docs/content/docs/dev/table/sql/show.md

```diff
@@ -740,6 +741,65 @@ show columns from orders not like '%_r';
 4 rows in set
 ```
 
+## SHOW PARTITIONS
+
+```sql
+SHOW PARTITIONS [[catalog_name.]database.]<table_name> [ PARTITION <partition_spec>]
+
+<partition_spec>:
+  (key1=val1, key2=val2, ...)
+```
+
+Show all partitions of the partitioned table with the given table name and optional partition clause.
+
+**PARTITION**
+Show all partitions of the partitioned table with the given table name and optional `PARTITION` clause, which are under the provided <partition_spec>.
```

Comment:
Seem there's a grammar issue it this sentence, I would like to change it to:
Show all the partitions which are under the provided <partition_spec>.